apiserver/db/user.go

```diff
@@ -24,6 +25,11 @@ func icosToUris(icos []string) []string {
 	return uris
 }
 
+func uriToIco(uri string) string {
```

Comment:
maybe we can return `(string, error)` here and check if prefix (`ico://sk/`) is valid IČO prefix
so this function is more reusable and does not expect only valid uris